Packaging.props

```diff
@@ -18,6 +18,8 @@
     <NativePackagePath>$(MSBuildThisFileDirectory)src/Native/pkg</NativePackagePath>
   </PropertyGroup>
 
+  <Import Project="pkg/baseline/baseline.props" />
```

Comment:
Can we make this conditional on existence? Again I'm thinking of other repo's like CLI or CoreCLR that might not have a unique baselining requirement.